vpr/src/draw/draw.cpp

```diff
@@ -202,13 +224,10 @@ const std::vector<ezgl::color> kelly_max_contrast_colors = {
     ezgl::color(43, 61, 38)     //olive green
 };
 
-ezgl::application::settings settings("/ezgl/main.ui",
-                                     "MainWindow",
-                                     "MainCanvas",
-                                     "org.verilogtorouting.vpr.PID" + std::to_string(vtr::get_pid()),
-                                     setup_default_ezgl_callbacks);
+ezgl::application::settings settings("/ezgl/main.ui", "MainWindow", "MainCanvas", "org.verilogtorouting.vpr.PID" + std::to_string(vtr::get_pid()), setup_default_ezgl_callbacks);
 ezgl::application application(settings);
 
+//bool mm_window = false;
```

Comment:
I think it is a debugging variable or an old one that need to be deleted at all (not just commented out).